src/FunctionLoader.cs

```diff
@@ -65,7 +65,7 @@ internal static void ClearLoadedFunctions()
         /// Setup the well known paths about the FunctionApp.
         /// This method is called only once during the code start.
         /// </summary>
-        internal static void SetupWellKnownPaths(FunctionLoadRequest request)
+        internal static void SetupWellKnownPaths(FunctionLoadRequest request, string managedDependenciesPath)
```

Comment:
If we do ```string managedDependenciesPath = null```, then we do not have to modify any code that calls into ```SetupWellKnownPaths```

Reply:
Currently, `null` can be passed from tests only. Making this a default would create an opportunity to forget passing this parameter in production code as well. I would rather have tests being very explicit about passing a `null` than catch this bug late.

Reply:
Sounds good.